scss/components/atoms/_button-unstyled.scss

```diff
@@ -0,0 +1,5 @@
+@charset 'UTF-8';
```

Comment:
New atom directory is not needed but might be helpful. Example of an item is an unstyled button.